.github/workflows/openjdk_14_build.yml

```diff
@@ -0,0 +1,117 @@
+name: openjdk_14_build
+on:
+  pull_request:
+  push:
+    branches: [ main, develop ]
+    tags: [ v* ]
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    container: gcr.io/swirlds-registry/cci-openjdk-ghr:14.0.2
```

Comment:
Why the hardcoded swirlds container? We should omit this and use the standard `actions/setup-java@v2` as a step instead.

Reply:
this is the container that circleci was using, I'll try the new one